Jenkinsfile

```diff
@@ -1,5 +1,12 @@
 import groovy.json.JsonOutput
 
+class JdbcJobDefinition {
+  String jdk
+  List params
+  String jobToRun
+  String runName
+}
+
 pipeline {
   // TODO Please migrate this to C7 as sfc-dev2 servers do not support c6 nodes
```

Comment:
we won't run on Centos 6 anymore?

Reply:
This is more complex. Let me describe my understanding.

Jobs are started on some kind of nodes. This nodes are specified in job definition which is in jenkins_utils. Currently we use node named `regular`. This is what this comment was about.
When started, job can perform anything. Our job runs some tests inside some containers. This containers may still use C6 (but this PR switches it off by default), but this is run inside specific Jenkins node and as far as I understand - it may be anything.